Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/v1/defaults.go

```diff
@@ -323,4 +324,70 @@ func defaultSecurityContextConstraints(scc *SecurityContextConstraints) {
 		scc.AllowEmptyDirVolumePlugin = new(bool)
 		*scc.AllowEmptyDirVolumePlugin = true
 	}
+
+	defaultSecurityContextConstraintsVolumes(scc)
+}
+
+// defaultSecurityContextConstraintsVolumes defaults the volume slice of the SCC.
+// The volumes slice will take precedence in settings if it is set.
```

Comment:
the takeaway is that the booleans always win. clients will have to set the booleans AND the volumes slice (or maybe omit the booleans if we make them pointers, but I think that breaks clients talking to older v1 servers)